docs/error-handling.md

```diff
@@ -0,0 +1,13 @@
+# Error handling
```

Comment:
I'm not sure I understand why we have two pages. They seem to be talking about the same thing? 🤔 

I think it would be better maintenance-wise to just have one. If you prefer keeping this `error-handling.md`, please make sure to update the table of contents in the [index](https://github.com/woocommerce/woocommerce-ios#-documentation) too. 😊

Reply:
Thanks for the review! I don't have a strong reason for the additional file, mostly visibility 😅  I agree that is easier to maintain with only referencing the existing `coding-style-guide.md`. Changed on https://github.com/woocommerce/woocommerce-ios/pull/8461/commits/387f6b0b7d39ff42e1c49db1a249f250e4420dc4

Enjoy your AFK!